mozilla/bedrock
# Revise UX/Behavior of Sticky Download Promo

## Description

This PR adds two revisions to the UX of the [sticky download promo](https://protocol.mozilla.org/patterns/molecules/sticky-promo.html):

1. When the footer is in the viewport, the promo box is hidden. 
2. The promo box is not displayed until a user scrolls on the page. 

TODO: 

- [x] Revise/remove tests that are no longer needed 

## Issue / Bugzilla link

#9717

## Testing

*Be sure to delete any `localhost` cookeis you may have*

1. Go to http://localhost:8000/en-US/firefox/new/
2. Scroll down the page some. 
3. Expected: Promo should appear
4. Scroll down to where the footer is in view. 
5. Expected: Promo should dissapear 
6. Scrioll back up to where the footer is no longer in view
7. Expected: Promo should reappear 

Notes: 
- I couldn't find a page that had the promo that was so short that the footer was already in view. It is assumed that the promo would not be displayed. 
- If you dismiss the promo, and continue to scroll (including into and out of the footer), it remains hidden. On refresh/reload, the promo is not loaded (per the current logic)
